### ISSUE TITLE
Tnagier International Zone

### ISSUE THREAD

**Issue by arandomperson5000** [simsulla/nwomod#1088]
Recently the Tangier International Zone has been added into CWE, I think it could be added into NWO too as it was an international zone that was part of neither Spanish or French Morocco, was completely independent of both entities and that Trieste another former international zone is represented as its own country in NWO. An event for the annexation of Tangier into Morocco will be created to accompany this.
https://en.wikipedia.org/wiki/Tangier_International_Zone

**Comment by simsulla** [comment]
Morocco does not have many events. Currently we don't even have the [Ifni War ](https://en.wikipedia.org/wiki/Ifni_War) in the mod. 

What events are CWE associated with it? Else, I don't see what yet another tiny core bring to game play. 

There are other cores that make more sense to me, like the [Rif Republic ](https://en.wikipedia.org/wiki/Rif_Republic)(the Rif wars was in the 20s)  that could be used to mod Berber-Arab conflict in Morocco. This would bring something to the game - as part of a Moroccan overhaul. 
(_I don't suggest adding it as just yet another core_)

**Comment by arandomperson5000** [comment]
I completely agree with you that Morocoo needs more events as there isn't that much Moroccan events such as the Ifni War you just mentioned in the above and the Western Sahara conflict that lasted from 1970 to 1991. I also completely agree that if makes more sense to put in the Rif Republic instead of Tangier to mod the Berber-Arab conflict in Morocco into NWO, the Rif Republic could also be released by Spain  in 1956 instead of giving back the lands its occupied as a protectorate in 1912 back to Morocco as most African countries follow old colonial borders and an independent Rif republic could be used to divert Moroccan attention away from its Western Sahara colony giving Spain a chance to try to preserve the last remnants of the once mighty Spanish Empire. Since there is an event for Nigeria for a hypothetical breakup of the country in the near future I think there should similar a event for Western Sahara in where it finds itself once again in conflict with Morocco in the near future at it appears of right now many residents of the Polisario controlled areas including soldiers are getting increasingly frustrated with the lack of political progress towards the issue of Western Saharan sovereignty making them want to fight Morocco.
https://news.vice.com/video/the-saharas-forgotten-war-full-length

**Comment by arandomperson5000** [comment]
Here is the event file for Morocco in CWE, the only events for Tangier are just an edited version of the event for the transfer of lands in the Spanish Protectorate back to Morocco in 1956 with Tangier added in.
https://github.com/settintotrieste/Victoria-2-Cold-War-Enhancement-Mod-CWE/blob/ec6f0ab9b73ec0950196008d82e804973bd96875/CWE/events/morocco.txt
